app/controllers/response_sets_controller.rb

```diff
@@ -46,6 +47,22 @@ def create
     end
   end
 
+  def transfer_table
+    @response_set.coded = @parent_set.coded
+    @response_set.oid = @parent_set.oid
+    @response_set.description = @parent_set.description
+    @response_set.name = @parent_set.name
+    @parent_set.responses.each do |r|
```

Comment:
@rdingwell I advised @kierk to implement it this way. The work that he was doing and what I am working on assumes that response set -> response is one-to-many as opposed to many-to-many. My gut feeling is that we punt on changing the relationship type until we get this functionality nailed down, but I'm willing to be talked out of that position.